src/feature/hibernate/hibernate.c

```diff
@@ -856,6 +856,10 @@ hibernate_begin(hibernate_state_t new_state, time_t now)
                "connections, and will shut down in %d seconds. Interrupt "
                "again to exit now.", options->ShutdownWaitLength);
     shutdown_time = time(NULL) + options->ShutdownWaitLength;
+#ifdef HAVE_SYSTEMD
+    sd_notifyf(0, "EXTEND_TIMEOUT_USEC=%lu",
+            options->ShutdownWaitLength * 1000000UL + 10000000);
```

Comment:
This could be rewritten as:
```suggestion
            options->ShutdownWaitLength * TOR_USEC_PER_SEC + 10 * TOR_USEC_PER_SEC);
```

But I wonder if you actually meant:
```suggestion
            options->ShutdownWaitLength * TOR_USEC_PER_SEC + 1 * TOR_USEC_PER_SEC);
```

Either way, it might be worth adding a comment to explain the extra second(s).

Reply:
Or even:
```suggestion
            (options->ShutdownWaitLength + 1) * TOR_USEC_PER_SEC);
```

Reply:
Yes, I deliberately changed the number from 10 to 1, when I said:

> But I wonder if you actually meant: